logs/zap.go

```diff
@@ -1,11 +1,57 @@
 package logs
 
 import (
+	"fmt"
+	cmiddleware "github.com/go-chi/chi/middleware"
```

Comment:
no need to use import alias here, you would still have to check the import directives for both "cmiddleware" and "middleware" to check what package is actually used